sdk/python/lib/pulumi/runtime/rpc.py

```diff
@@ -1221,7 +1228,8 @@ def resolve_outputs(
         # pylint: disable=C3001
         translate_to_pass = lambda prop: prop
 
-    for key, value in deserialize_properties(outputs, keep_unknowns).items():
+    deserialized, _ = deserialize_properties(outputs, keep_unknowns)
```

Comment:
Do we need to worry that these are now returning unwrapped rpc_secrets? While before it would have still be wrapped, and probably handled later in some way.

Reply:
Looking at where `deserialize_properties` currently is being used, I think it's fine. 

In this specific place though it is a bug 😅